### DISCUSSION TITLE
Is Swiftcord still alive?

### DISCUSSION THREAD

**Discussion by freepicheep** [cryptoAlgorithm/Swiftcord#139] [General]
I'm just curious if you are planning on moving this project forward, or if you are going to be done with it.

**Comment by cryptoAlgorithm** [comment]
It definitely is - Check out the Discord server [here](https://discord.gg/he7n6MGDXS) and take a look at the [`devel`](https://github.com/SwiftcordApp/Swiftcord/tree/devel) branch where most of the development currently takes place!

**Reply by freepicheep** [reply]
Wonderful! Thanks for letting me know. I love native apps and really want to learn Swift. I've been following the development of CodeEdit for a while, and I'll probably just tag along with this project until I get some time to learn Swift and can contribute. I joined the Discord server and will take a look at the `devel` branch.

**Reply by cryptoAlgorithm** [reply]
No problem! All the best for your Swift journey!

**Reply by cryptoAlgorithm** [reply]
There's been a big update to how things are going, take a look at #189!

**Comment by MrPaschenko** [comment]
Most realistically, no
